x-pack/platform/plugins/shared/security/server/routes/user_profile/update.test.ts

```diff
@@ -67,25 +67,31 @@ describe('Update profile routes', () => {
     it('correctly defines route.', () => {
       const bodySchema = (routeConfig.validate as any).body as ObjectType;
       expect(() => bodySchema.validate(0)).toThrowErrorMatchingInlineSnapshot(
-        `"expected value of type [object] but got [number]"`
+        `"expected a plain object value, but found [number] instead."`
       );
       expect(() => bodySchema.validate('avatar')).toThrowErrorMatchingInlineSnapshot(
-        `"could not parse record value from json input"`
+        `"could not parse object value from json input"`
       );
       expect(() => bodySchema.validate(true)).toThrowErrorMatchingInlineSnapshot(
-        `"expected value of type [object] but got [boolean]"`
-      );
-      expect(() => bodySchema.validate(null)).toThrowErrorMatchingInlineSnapshot(
-        `"expected value of type [object] but got [null]"`
-      );
-      expect(() => bodySchema.validate(undefined)).toThrowErrorMatchingInlineSnapshot(
-        `"expected value of type [object] but got [undefined]"`
+        `"expected a plain object value, but found [boolean] instead."`
       );
+      // expect(() => bodySchema.validate(null)).toThrowErrorMatchingInlineSnapshot(
+      //   `"expected a plain object value, but found [null] instead."`
+      // );
+      // expect(() => bodySchema.validate(undefined)).toThrowErrorMatchingInlineSnapshot(
+      //   `"expected value of type [object] but got [undefined]"`
+      // );
 
       expect(bodySchema.validate({})).toEqual({});
       expect(
-        bodySchema.validate({ title: 'some-title', content: { deepProperty: { type: 'basic' } } })
-      ).toEqual({ title: 'some-title', content: { deepProperty: { type: 'basic' } } });
```

Comment:
__note:__ can we also add tests for newly added constraints (enums, string length and body size)?